lib/haml_lint/tree/tag_node.rb

```diff
@@ -198,5 +206,17 @@ def script
     def text
       (@value[:value] if @value[:parse]) || ''
     end
+
+    private
+
+    def parsed_attributes
+      HamlLint::RubyParser.new.parse(hash_attributes_source)
```

Comment:
Seems like there can be some additional modeling/extraction done around the parser, but I wasn't sure how far to take it. Open to suggestions.